service-tests/jetbrains.js

```diff
@@ -3,31 +3,53 @@
 const Joi = require('joi');
 const ServiceTester = require('./runner/service-tester');
 const { isMetric } = require('./helpers/validators');
+const { isSemver } = require('./helpers/validators');
 
-const t = new ServiceTester({ id: 'jetbrains', title: 'JetBrains plugin' });
+const t = new ServiceTester({ id: 'jetbrains', title: 'JetBrains plugin/version' });
```

Comment:
Rather than keep updating this, you can just call it `'JetBrains'`.

Reply:
Done.